.circleci/run-tests.sh

```diff
@@ -65,7 +65,7 @@ TIMEOUT="timeout --kill-after 1m 15m"
 # Send the output directly to a file because transporting the binary subunit2
 # via tox and then scraping it out is hideous and failure prone.
 export SUBUNITREPORTER_OUTPUT_PATH="${SUBUNIT2}"
-export TAHOE_LAFS_TRIAL_ARGS="--reporter=subunitv2-file --rterrors"
+export TAHOE_LAFS_TRIAL_ARGS="${TAHOE_LAFS_TRIAL_ARGS:---reporter=subunitv2-file --rterrors}"
```

Comment:
Does this help with one of the CI configurations or is it to make it possible to run this script manually, outside of CI?  It looks like all the CI jobs still set this, so it will always be defined?

Reply:
This is intended to make `TAHOE_LAFS_TRIAL_ARGS` settable from outside this script, so that I can unset `--rterrors` to workaround the conflict between that and `--reporter` under Python 3. Without this, any setting external to this script is overridden here. No?

```bash
#!/usr/bin/env bash
export FOO="--foo"
export BAR="${BAR:---bar}"
echo "baz $FOO $BAR"
```

```
$ ./test
baz --foo --bar
$ FOO=--cheese BAR=--bread ./test 
baz --foo --bread
$
```

> It looks like all the CI jobs still set this

I don't find `TAHOE_LAFS_TRIAL_ARGS` mentioned anywhere (else) in `.circleci/config.yml`. Am I looking in the wrong place? 🤔

Reply:
Ah you're right.  Thanks for laying it out.